### ISSUE TITLE
Feature request: volume control

### ISSUE THREAD

**Issue by byggmastar** [Rudd-O/hassmpris_agent#4]
Could you make it possible to adjust(increase and decrease) volume also via home assistant? Pretty please.

I love it. Got an Ikea smart light switch to control HBO max on chrome. <3

**Comment by Rudd-O** [comment]
Yeah I think this should be codeable.  What device are you watching Chrome on?  I currently control volume via my receiver (or TV if the receiver is off).

**Comment by byggmastar** [comment]
Wonderful. I'm running Chrome on Ubuntu 23.10 on computer with some second and speakers I bought second hand ages ago.
